Update the makefile and build-locally page (#202)
canonical/ubuntu-server-documentation: contributing/build-locally.rst

will give update the preview to show your changes (or warn you if something has
gone horribly wrong!).

If you are building locally on an Ubuntu Cloud VM or a container, you may experience issues accessing the page from your host's 
browser. To resolve this, add the export variable to your shell by running the following command:

.. code-block::

   export SPHINX_HOST=0.0.0.0

> Running the `make run` command should then work as expected.

Reviewer: ```suggestion
Running the ``make run`` command should then work as expected.
```

will give update the preview to show your changes (or warn you if something has
gone horribly wrong!).

If you are building locally on an Ubuntu Cloud VM or a container, you may experience issues accessing the page from your host's 
browser. To resolve this, add the export variable to your shell by running the following command:

> `export SPHINX_HOST=0.0.0.0`

Reviewer: ```suggestion
.. code-block::

   export SPHINX_HOST=0.0.0.0

```

Author: It's ok

Running the `make run` command should then work as expected.

> .. code-block::

Author: Alright

will give update the preview to show your changes (or warn you if something has
gone horribly wrong!).

If you are building locally on an Ubuntu Cloud VM or a container, you may experience issues accessing the page from your host's 

> browser. To resolve this, export the variable with `export SPHINX_HOST=0.0.0.0` before executing `make run`.

Reviewer: ```suggestion
browser. To resolve this, add the export variable to your shell by running the following command:

```bash
export SPHINX_HOST=0.0.0.0
```

Running the `make run` command should then work as expected.
```

Reviewer: Just to make it explicit what we're talking about :)

Author: Nice!
